slack_etiquette.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Slack Etiquette
+permalink: /about/slacketiquette
+published: true
+---
+
+
+# Slack Etiquette
+### Aka meming and messaging responsibly - msoe-sse.slack.com
+> Few people want constant phone buzzes 
+
+## 1. Unless you are on the executive board, do NOT use @channel in #general or #random under any circumstances
+> If you do have something of importance, please message _Brady Cassada_, the Communications Coordinator

```

Comment:
Let's be more specific here--what constitutes something of importance? When is it better to slack Brady than just post in the #general channel?